### ISSUE TITLE
Issue with DM menu

### ISSUE THREAD

**Issue by PedroCavaleiro** [PatrykBielanin/discord-cyberpunk2077-theme#13]
When the DM menu is selected a yellow background is visible the same doesn't happen to the servers which leaves me to believe that this is a bug

<img width="313" alt="Screenshot 2024-04-22 at 01 00 57" src="https://github.com/PatrykBielanin/discord-cyberpunk2077-theme/assets/1993871/d73cfddf-61d5-4fd9-9687-b3a91e5f1acb">

**Comment by Silver-Fox** [comment]
Can confirm I'm having the same problem as well

**Comment by Maicouh** [comment]
Same here

**Comment by EinarsNG** [comment]
Adding Custom CSS worked for me:
```
.selected__501f5 {
  background-color: transparent !important;
}
```

**Comment by Silver-Fox** [comment]
> Adding Custom CSS worked for me:
> 
> ```
> .selected__501f5 {
>   background-color: transparent !important;
> }
> ```

worked like a charm, thanks my man.

**Comment by PatrykBielanin** [comment]
Fixed with #16
